### PR TITLE
Create frontend of versioning

### PR DESCRIPTION
**Pull request by tin0d0**
- add versioning button
- add box to contain revisions and commits
- add button and input for commits

### CODE REVIEW

**Review by max-friedman — APPROVED** [review]
lgtm

File: project/src/main/webapp/document.jsp [hunk 1]
```diff
@@ -102,6 +158,7 @@
     </div>
     <div class="toolbar">
       <toolbar-component onclick="changeTheme()"></toolbar-component>
+      <button class="verion-btn" onclick="showVersioning()">Versioning</button>
```
Comment (g-luo) [inline project/src/main/webapp/document.jsp:161]: should the class be "version-btn"?

File: project/src/main/webapp/document.jsp [hunk 2]
```diff
@@ -122,7 +179,27 @@
...
+      </div>
+      <div class="versionHeader">
+        <div class="revisions">
+          <button class="text-btn" id="revisions-button"> Revisions </button>
+        </div>
+        <div class="commits">
+          <button class="text-btn" id="commits-button"> Commits </button>
+        </div>
+      </div>
+      <div class="commitButton three-width">
```
Comment (g-luo) [inline project/src/main/webapp/document.jsp:194]: maybe rename "three-width" to "three-quarters-width" for clarity

Comment (g-luo) [reply project/src/main/webapp/document.jsp:194]: or maybe "almost-full-width"

**Review by g-luo — APPROVED** [review]
lgtm

**Review by frankpzn — APPROVED** [review]

**Review by HidyHan — APPROVED** [review]

**Review by jileiwang-google — APPROVED** [review]
LGTM

**Pull request merged**